XFCorePostProcessor.Tasks/FixXFCoreAssembly.cs

```diff
@@ -32,6 +34,10 @@ public override bool Execute()
 			using (var assemblyDefinition = AssemblyDefinition.ReadAssembly(Assembly, new ReaderParameters { AssemblyResolver = resolver, ReadWrite = true, ReadSymbols = true })) {
 				var resourceLoader = assemblyDefinition.MainModule.GetType("Xamarin.Forms.Internals.ResourceLoader");
 				var module = assemblyDefinition.MainModule;
+				if(resourceLoader.GetMethods().Count(md=>md.Name == "get_ResourceProvider") > 1) {
```

Comment:
```suggestion
				if (resourceLoader.GetMethods().Count(md => md.Name == "get_ResourceProvider") > 1) {
```

Reply:
Soooo...wouldn't we want to stop this before it added the second one?

`.Any()` would be faster, but that won't work if we expect there to be two methods.

Reply:
or `.Skip(1).Any()`. not really important, we do not run on device here